lib/ex_aws/sqs.ex

```diff
@@ -328,6 +332,21 @@ defmodule ExAws.SQS do
     Map.put(%{}, key, format_param_key(attribute))
   end
 
+  defp format_message_attribute({attribute, index}) do
+    %{"MessageAttributeName.#{index + 1}" => attribute}
```

Comment:
Yup let's do that, and change the type signature for these to `String.Chars.t` cause that'll cover anything handled by `to_string`